gettsim/benefits/kiz.py

```diff
@@ -96,6 +99,18 @@ def kiz(household, tb):
     return household
 
 
+def calc_min_income_kiz(household, tb):
+    # Are there kids in the household
+    if len(household[household["child"]]) > 0:
```

Comment:
Let us please not rely on this actually being a Boolean value. Or have a check right in the begininng of the code.

Reply:
I saw somewhere else that we want to have some kind of identifier saying: this individual is a child of the other. So we will change all these functions anyway. Therefore I think we should stick with a test for now!

Reply:
agreed to leave it for now (this holds almost all of the changes in this PR).
But `household['child'].sum()` would work for both types and is shorter.